testsuite/tests/typing-modules/aliases.ml

```diff
@@ -640,6 +642,8 @@ module rec Bad : A = Bad;;
 module type Alias = sig module N : sig  end module M = N end
 module F : functor (X : sig  end) -> sig type t end
 Line _:
+  module C = Char;;
+
```

Comment:
This `module C = Char` is the first line of the file, because I suppose there is no location and so line is 1 and I am not sure about the characters. It would be preferable to print nothing than to print the first line.

Ideally, the expectation would be clearer about the complete lack of position, so developers are more likely to fix them and reviewers more likely to point them out.

Reply:
Oh, the characters are probably 0-0, which is why nothing is underlined.

Reply:
Actually, I think that grepping for `Line _:` is enough to detect fishy locations: the characters are not printed when they look non-sensical, and I think this coincides with what you want to detect. In that case, I would suggest not changing anything in the PR specifically: (1) it's an orthogonal concern and (2) the reasonable need that you suggested can already be satisfied in the current state (if my guess is right that `Line _:` coincides with the situation where you think we should "be clearer").

Reply:
Oh ok, this is the one place where something fishy is going on.
It's actually more fishy than I first thought, because ocaml/expect_test give one error, and ocamlc/ocamlopt give a different looking error, which I think is a better version of the same error,  with a position in particular. 

In any case, it still seems to me that the whole show_code_at_location call should be below a `if startchar >= 0`, to avoid printing unrelated code.

Reply:
`expect_test` is using `Toploop` which is the `ocaml` (toplevel) path, maybe the discrepancy can be reproduced there? You are right that it's interesting enough to investigate, I'll try to have a look.